resources/scss/_common.scss

```diff
@@ -0,0 +1,321 @@
+@import './_variables.scss';
+
+@mixin flex {
+  display: flex;
+}
+
+@mixin flex-row {
+  @include flex;
+  flex: 1;
+}
+
+@mixin flex-column {
+  @include flex;
+  flex: 1;
+  flex-direction: column;
+}
+
+@mixin flex-row-reverse {
+  @include flex;
+  flex-direction: row-reverse;
+}
+
+@mixin flex-column-reverse {
+  @include flex;
+  flex-direction: column-reverse;
+}
+
+@mixin break-point($point) {
+  @if $point == tablet {
+    @media only screen and (min-width: 480px) {
+      @content;
+    }
+  } @else if $point == desktop {
+    @media only screen and (min-width: 1024px) {
+      @content;
+    }
+  }
+}
+
+@mixin app-header-padding-changes {
+  @include break-point(desktop) {
+    padding-left: $app-padding-horizontal;
+    padding-right: $app-padding-horizontal;
+  }
+}
+
+@keyframes fadein {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes fadeout {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+}
+
+.flex {
+  @include flex;
+}
+
+.flex-row {
+  @include flex-row;
+}
+
+.flex-column {
+  @include flex-column;
+}
+
+.flex-row-reverse {
+  @include flex-row-reverse;
+}
+
+.flex-column-reverse {
+  @include flex-column-reverse;
+}
+
+.flex-center {
+  align-items: center;
+  justify-content: center;
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+  color: $text-color-primary;
+}
+
+a {
+  color: $text-color-primary;
+  text-decoration: none;
+  user-select: none;
+}
+
+html,
+body {
+  margin: 0 auto;
+  font-family: 'Dosis';
+  font-weight: $app-font-weight;
+  box-sizing: border-box;
+  font-size: 0.75rem;
+  @include break-point(tablet) {
+    font-size: 1rem;
+  }
+}
+
+.btn {
+  background: $background-color-secondary;
+  border: 0;
+  padding: $app-btn-padding;
+  color: $text-color-secondary;
+  font-family: 'Dosis';
+  text-align: center;
+  font-size: 1rem;
+}
+
+.nav {
+  display: flex;
+  height: $header-height;
+  box-shadow: 0 8px 6px -3px #eeeeee;
```

Comment:
Define different box shadow values in variables, and use rem and define color in variables.